lib/stream.js

```diff
@@ -86,14 +106,19 @@ const applyEncoding = async (stream, maxBuffer, encoding) => {
 	return buffer.toString(encoding);
 };
 
+// Handle any errors thrown by the iterable passed to the `stdin` option, if any.
+// We do not consume nor wait on that stream though, since it could potentially be infinite (like `process.stdin` in an interactive TTY).
+const throwOnStreamsError = streams => streams.filter(Boolean).map(stream => once(stream, ''));
```

Comment:
`once()` throws on `error` event, regardless of the event passed as second argument (in that case, a dummy empty string).

Another way of writing this (longer, but maybe clearer?) would be:

```js
const [error] = await once(stream, 'error')
throw error
```